NHLBI_BioData_Catalyst/R/R_lib/ORCHID_requirements.R

```diff
@@ -1,5 +1,7 @@
 install.packages("versions", dependencies=TRUE)
```

Comment:
Note for SB testing of the ORCHID_COVID19.ipynb: I had to run the `source("R_lib/ORCHID_requirements.R")` line because there were several issues with extra packages (forcats, tableby, etc). After running that line with specific ORCHID requirements, it worked. Recommend uncommenting this line in the notebook.

Reply:
seconding the fact that it worked with this line, however it took a long time (~5-7 minutes to finish the chunk). Is there a way to check the dependencies 1st (like in the python version) so the user knows if they have to run this chunk or not?

Reply:
Resolved in [2813ad727d5a7217df608e10c0b36913cef55203](https://github.com/hms-dbmi/Access-to-Data-using-PIC-SURE-API/commit/2813ad727d5a7217df608e10c0b36913cef55203)